addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/ExtensionParamMiner.java

```diff
@@ -34,7 +34,7 @@
 public class ExtensionParamMiner extends ExtensionAdaptor {
 
     public static final String NAME = "ExtensionParamMiner";
```

Comment:
Here too

Reply:
I think the code/classes will be renamed in another PR.

Reply:
If you are changing the name change the class as well.

Reply:
Anyway.